Vagrantfile

```diff
@@ -0,0 +1,48 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+
+  config.vm.box = "generic/ubuntu1804"
+  config.vm.network "forwarded_port", guest: 8000, host: 8000
+  config.vm.network "public_network",
+    use_dhcp_assigned_default_route: true
+  config.vm.synced_folder ".", "/vagrant"
+  config.vm.provision :shell, inline: <<-SHELL
+#!/bin/bash
+
+set -euox pipefail
+
+echo "=== bootstrap ==="
+
+apt-get update
+apt-get install -y docker.io curl make
```

Comment:
Can you use the newest version of docker instead?
https://docs.docker.com/install/linux/docker-ce/ubuntu/

Reply:
sure.

with the latest changes:

vagrant@ubuntu1804:~$ sudo docker version
Client: Docker Engine - Community
 Version:           19.03.0
 API version:       1.40
 Go version:        go1.12.5
 Git commit:        aeac949
 Built:             Wed Jul 17 18:15:07 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.0
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.5
  Git commit:       aeac949
  Built:            Wed Jul 17 18:13:43 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.6
  GitCommit:        894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc:
  Version:          1.0.0-rc8
  GitCommit:        425e105d5a03fabd737a126ad93d62a9eeede87f
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683